SETUP.md

```diff
@@ -101,65 +101,63 @@ To install the environment:
-
-```bash
-#!/bin/sh
-unset PYSPARK_PYTHON
-unset PYSPARK_DRIVER_PYTHON
-export SPARK_HOME=$SPARK_HOME_BACKUP
-unset SPARK_HOME_BACKUP
-```
-
-</details>
-
-
-<details>
-<summary><strong><em>Windows</em></strong></summary>
-
-To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#windows).
-Assuming that we have installed the environment in `c:/anaconda/envs/reco_pyspark`,
-create the file `c:/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.bat` and add:
-
-    @echo off
-    set PYSPARK_PYTHON=/anaconda/envs/reco_full/python.exe
-    set PYSPARK_DRIVER_PYTHON=/anaconda/envs/reco_full/python.exe
-    set SPARK_HOME_BACKUP=%SPARK_HOME%
-    set SPARK_HOME=
-    set PYTHONPATH_BACKUP=%PYTHONPATH%
-    set PYTHONPATH=
-
-This will export the variables every time we do `conda activate reco_pyspark`.
-To unset these variables when we deactivate the environment,
-create the file `/anaconda/envs/reco_pyspark/etc/conda/deactivate.d/env_vars.bat` and add:
-
-    @echo off
-    set PYSPARK_PYTHON=
-    set PYSPARK_DRIVER_PYTHON=
-    set SPARK_HOME=%SPARK_HOME_BACKUP%
-    set SPARK_HOME_BACKUP=
-    set PYTHONPATH=%PYTHONPATH_BACKUP%
-    set PYTHONPATH_BACKUP=
-
-</details>
+>
+> <details>
+> <summary><strong><em>Linux or MacOS</em></strong></summary>
+>
+> To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#macos-and-linux).
+> Assuming that we have installed the environment in `/anaconda/envs/reco_pyspark`,
+> create the file `/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.sh` and add:
+>
+>     ```bash
+>     #!/bin/sh
+>     export PYSPARK_PYTHON=/anaconda/envs/reco_pyspark/bin/python
+>     export PYSPARK_DRIVER_PYTHON=/anaconda/envs/reco_pyspark/bin/python
+>     export SPARK_HOME_BACKUP=$SPARK_HOME
+>     unset SPARK_HOME
+>     ```
+>
+> This will export the variables every time we do `conda activate reco_pyspark`.
+> To unset these variables when we deactivate the environment,
+> create the file `/anaconda/envs/reco_pyspark/etc/conda/deactivate.d/env_vars.sh` and add:
+>
+>     ```bash
+>     #!/bin/sh
+>     unset PYSPARK_PYTHON
+>     unset PYSPARK_DRIVER_PYTHON
+>     export SPARK_HOME=$SPARK_HOME_BACKUP
+>     unset SPARK_HOME_BACKUP
+>     ```
+> 
+> </details>
+>
+> <details><summary><strong><em>Windows</em></strong></summary>
+> 
+> To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#windows).
+> Assuming that we have installed the environment in `c:/anaconda/envs/reco_pyspark`,
+> create the file `c:/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.bat` and add:
+> 
+>     @echo off
+>     set PYSPARK_PYTHON=/anaconda/envs/reco_full/python.exe
```

Comment:
Does these a c:/ in front?

Reply:
yes, will add here too. without c: works if anaconda is installed on the same drive as Windows', but if not, should specify I guess.